build.cmd

```diff
@@ -104,7 +104,6 @@ set _binclashLoggerDll=%~dp0Tools\net45\Microsoft.DotNet.Build.Tasks.dll
 set _binclashlog=%~dp0binclash.log
 set _buildprefix=echo
 set _buildpostfix=^> "%_buildlog%"
-
```

Comment:
NIT: Normally I prefer an empty line for clearance between defining vars and doing work, but if this is to sync with the build.cmd in corefx, I can live with it 😄.